projects/exui-common-lib/src/lib/services/loading/loading.service.ts

```diff
@@ -1,14 +1,20 @@
-import { Injectable } from "@angular/core";
-import { BehaviorSubject, Observable } from "rxjs";
-import { distinctUntilChanged } from "rxjs/operators";
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { distinctUntilChanged } from 'rxjs/operators';
 
 
 @Injectable({
   providedIn: 'root'
 })
+export abstract class HasLoadingState {
+  public get isLoading(): Observable<boolean> {
+    return;
+  }
```

Comment:
that isn't returning an observable boolean....

Reply:
this is exisiting code I just fixed the lint issue, but anyway I have now changed it to abstract method